src/Integration.Vsix/source.extension.vsixmanifest

```diff
@@ -2,7 +2,7 @@
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
     <!-- Note: keep the version number in sync with AssemblyInfo.Shared.cs -->
-    <Identity Id="SonarLint.8c442ec8-0208-4840-b83f-acd24f41acf7" Version="2.13.0.0" Language="en-US" Publisher="SonarSource SA" />
+    <Identity Id="SonarLint.8c442ec8-0208-4840-b83f-acd24f41acf7" Version="2.13.0.2" Language="en-US" Publisher="SonarSource SA" />
```

Comment:
I restored it now, thankfully I don't get the weird error anymore that I did before. Good, but strange...